src/pages/profile/model/store.ts

```diff
@@ -0,0 +1,6 @@
+import { createStore } from 'effector';
+import { User, getCurrentUserFx } from 'entities/session/model';
```

Comment:
You're exporting the guts from the model. imagine the model as one entity, one thing that can't be separated. We are only using methods of the model